website/institutions/model.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+import urlparse
+
+from django.core.urlresolvers import reverse
+
+from modularodm import fields
+from modularodm.validators import URLValidator
+
+from website import settings
+from framework.mongo import StoredObject
+
+class Institution(StoredObject):
+
+    _id = fields.StringField(index=True, unique=True, primary=True)
+    name = fields.StringField(required=True)
+    logo_name = fields.StringField(required=True)
+    auth_url = fields.StringField(required=False, validate=URLValidator())
+
+    def __repr__(self):
+        return '<Institution ({}) with id \'{}\'>'.format(self.name, self._id)
+
+    @property
+    def pk(self):
+        return self._id
+
+    @property
+    def absolute_url(self):
+        return urlparse.urljoin(settings.DOMAIN, self.url)
+
+    @property
+    def url(self):
```

Comment:
This seems incorrect; `Institutions` aren't GUID objects, so they shouldn't have a GUID short URL. Is this actually used anywhere? Can it be removed?